dbms/programs/server/Server.cpp

```diff
@@ -556,8 +556,26 @@ int Server::main(const std::vector<std::string> & /*args*/)
     try
     {
         loadMetadataSystem(*global_context);
-        /// After attaching system databases we can initialize system log.
-        global_context->initializeSystemLogs();
+
+        if (!config().getBool("system_tables_lazy_load", true))
```

Comment:
@alexey-milovidov @DeifyTheGod maybe it worth thinking making it false by default?